### PR TITLE
doc(README) : add prerequisites

### PR DESCRIPTION
**Pull request by adriens**
# :grey_question: About

While trying to play with `ollama-python` on a brand new env, I found out that following the current steps of the README do not take in account that `ollama` must be up and running (ie. that it's not `ollama-python` role).

# :dart: Objective

The purpose of this PR is to make the _Getting started_ even easier to people can just copy/paste and enjoy :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
...
+## Prerequisites
+
+Prior to be able to use the next steps, you need to have `ollama` [installed](https://ollama.ai/download), up and running:
+
+```sh
+curl https://ollama.ai/install.sh | sh
+ollama serve
+```
+
+Next you can go ahead with `ollapa-python`.
```
Comment (mikeesto) [inline README.md:14]:
```suggestion
Next you can go ahead with `ollama-python`.
```

**Comment by mak448a** [comment]
This would close #29.

**Comment by connor-makowski** [comment]
Worth mentioning that there are a few ways to do this. For example, you might also add something like:

```
## Local Ollama Setup
You need to have a local ollama server running to be able to continue. To do this:

- Download: https://ollama.com/
- Run an LLM: https://ollama.com/library
    - Example: `ollama run llama2`
    - Example: `ollama run llama2:70b`
```

**Comment by adriens** [comment]
:pray: @connor-makowski

**Comment by haesleinhuepf** [comment]
Big thanks @adriens

**Comment by haesleinhuepf** [comment]
Dear maintainers,

this is a very useful documentation update. Consider merging it. 😉

**Review by mxyng — APPROVED** [review]

**Comment by adriens** [comment]
:partying_face:

**Comment by adriens** [comment]
[Tweet](https://twitter.com/rastadidi/status/1789391046352249322)

image

**Comment by connor-makowski** [comment]
Referencing this to: https://github.com/ollama/ollama-python/pull/155 as an extension for adding clarity to the getting started process.

**Pull request merged**